### PR TITLE
Add links to email products, per Jellyfish: INFODEV-2511

### PR DESCRIPTION
**Pull request by RackerKate**

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-dfdf9b9897/how-to/). :bow:

**Comment by RackerKate** [comment]
@cyrichardson Please review.

**Comment by cyrichardson** [comment]
Reviewing this Jellyfish change inline because of it's size.

**Comment by cyrichardson** [comment]
@RackerKate Just saw a small change to make - Microsoft Exchange - capitalize the E.

File: content/rackspace-email/index.md [hunk 1]
```diff
@@ -3,14 +3,18 @@ title: Rackspace Email
...
 permalink: /rackspace-email/
 product: Rackspace Email
 product_url: rackspace-email
 ---
 
-Welcome to Rackspace Email! Use this brief step-by-step guide to help you get started. See [Rackspace Office 365](/how-to/office-365) or [Microsoft exchange](/how-to/exchange) for help with those products.
+Use this brief step-by-step guide to help you get started with [Rackspace
+Email](https://www.rackspace.com/email-hosting) and [Rackspace Email
+Webmail](https://www.rackspace.com/email-hosting/webmail). See [Rackspace
+Office 365](/how-to/office-365) or [Microsoft exchange](/how-to/exchange) for
```
Comment (cyrichardson) [inline content/rackspace-email/index.md]: Microsoft Exchange

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-a15426cc3d/how-to/)

**Pull request merged**